apiml-security-common/src/main/java/org/zowe/apiml/security/common/auth/saf/SafResourceAccessEndpoint.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package org.zowe.apiml.security.common.auth.saf;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.client.RestTemplate;
+
+@RequiredArgsConstructor
+public class SafResourceAccessEndpoint implements SafResourceAccessVerifying {
+
+    @SuppressWarnings("squid:S1075")
+    private static final String PATH_VRIABLE_SUFFIX = "/{userId}/{class}/{entity}/{level}";
+
+    @Value("${apiml.security.safEndpoint.url}")
+    private String endpointUrl;
+
+    private final RestTemplate restTemplate;
```

Comment:
I am lazy to check the code myself so I am just asking. Is it protected by HTTPS? I somehow thought there are multiple RestTemplate beans.

Reply:
I think at this stage, http or https depends on the type of endpoint that will serve the API. Not on the consumer. I think it can stay like this until further refinement is needed.